experiment.py

```diff
@@ -0,0 +1,101 @@
+import alignment
+import argparse
+import cv2 as cv
+import matplotlib.pyplot as plt
+import numpy as np
+import statistics
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("path_to_UAV_key_points", type=str, nargs=1)
+parser.add_argument("path_to_satellite_key_points", type=str, nargs=1)
+parser.add_argument("path_to_UAV_image", type=str, nargs=1)
+parser.add_argument("path_to_satellite_image", type=str, nargs=1)
+parser.add_argument("path_to_meta_data", type=str, nargs=1)
+
+# parse arguments
+args = parser.parse_args()
+print(args.path_to_UAV_key_points)
+# read all key points for UAV and satellite images
+key_points_aerial = alignment.read_key_points_from_file(args.path_to_UAV_key_points[0])
+key_points_satellite = alignment.read_key_points_from_file(
+    args.path_to_satellite_key_points[0]
+)
+
+if (
+    len(key_points_aerial) < 4
+    or len(key_points_satellite) < 4
+    or len(key_points_aerial) != len(key_points_satellite)
+):
+    print("Sorry, there sre too few correspondences to count transformation")
+
+aerial_image = cv.imread(args.path_to_UAV_image[0])
+satellite_image = cv.imread(args.path_to_satellite_image[0])
+
+# read pixel coordinates of UAV image to count its (lat, lon) coordinates, read satellite image corners' coordinates
+with open(args.path_to_meta_data[0], "r") as f:
+    lat_left, lon_left = map(float, f.readline().split())
+    lat_right, lon_right = map(float, f.readline().split())
+    X, Y = map(int, f.readline().split())
+
+# find and draw matches
+pair_indexes = [j for j in range(len(key_points_aerial))]
+matches = alignment.find_matches(pair_indexes)
```

Comment:
```suggestion
matches = [cv.DMatch(i, i, 1) for i in range(len(key_points_aerial))]
```
And you don't need `find_matches` then